website/docs/r/cloud_provider_snapshot_backup_policy.html.markdown

```diff
@@ -72,6 +75,197 @@ resource "mongodbatlas_cloud_provider_snapshot_backup_policy" "test" {
   }
 }
 ```
+# Examples Modifying Polices
+When Cloud Backup is enabled for a cluster MongoDB Atlas automatically creates a default Cloud Backup schedule for the cluster with four policy items; hourly, daily, weekly, and monthly. Because of this default creation this provider automatically saves the Cloud Backup Snapshot Policy into the Terraform state. If the default works well for you then you do not need to do anything other than create a cluster with Cloud Backup enabled and your Terraform state will have this information if you need it. However,  if you want the policy to be different than the default simply follow the next examples.
+
+## Example Usage - Create a Cluster and Modify the 4 Default Policies Simultaneously
+This cluster has already been created and is here as an example
```

Comment:
The headings and the examples are a bit mixed up.  Simultaneously means "at the same time" so this heading and example don't match.  The heading as shown here would mean "you create a cluster and change that clusters default backup policy in the same Terraform apply run" - that is the first example on lines 15-77.  So I think the solve here is to use this heading for the first example and then remove this one.